### PR TITLE
Chore/update alertbuilder

### PR DESCRIPTION
**Pull request by OksanaFedorchuk**
Added AlertWrapperModifier, which creates alerts using new native alert view modifier.

### CODE REVIEW

File: UDF/View/AlertStyleUpd.swift [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        self.message = message
+        self.actions = actions()
+    }
+    
+    @resultBuilder
+    public struct AlertActionBuilder {
+        public static func buildBlock(_ actions: AlertAction...) -> [AlertAction] {
+            actions.map { $0 }
+        }
+    }
```
Comment (Maks-Jago) [inline UDF/View/AlertStyleUpd.swift]: Додай ще 
- buildArray
- buildEither(first
- buildEither(second
- buildLimitedAvailability
- buildExpression(_ expression: Void)

Comment (OksanaFedorchuk) [reply UDF/View/AlertStyleUpd.swift]: додала

File: UDF/View/AlertStyleUpd.swift [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        static func buildLimitedAvailability(_ components: [AlertAction]) -> [AlertAction] {
+            components
+        }
+        
+        public static func buildBlock(_ components: [AlertAction]...) -> [AlertAction] {
+            components.flatMap { $0 }
+        }
+        
+        public static func buildArray(_ components: [[AlertAction]]) -> [AlertAction] {
+            Array(components.joined())
```
Comment (Maks-Jago) [inline UDF/View/AlertStyleUpd.swift]: joined не підійде. 
components.flatMap

Comment (OksanaFedorchuk) [reply UDF/View/AlertStyleUpd.swift]: змінила

**Pull request merged**